qa/tasks/vstart_runner.py

```diff
@@ -268,9 +268,19 @@ class LocalRemote(object):
 
     def __init__(self):
         self.name = "local"
-        self.hostname = "localhost"
+        self._hostname = "localhost"
```

Comment:
please call
```py
super.__init__(...)
```
and if you want to skip it on purpose, might want to comment here to explain why.

Reply:
Most classes don't call init() of their parent class since a lot of setup needed for teuthology is not needed for vstart_runner. I've added this info in the comment as well.